tests/integration/utils.py

```diff
@@ -164,6 +144,39 @@ def _get_authentication_params(self):
         return {"auth": None}
 
 
+def download_archive(download_url, file_name_tar):
```

Comment:
can we change this variable name (`file_name_tar`)? The archive could also be a zip file here, right?